_test_unstructured_client/test_utils_retries.py

```diff
@@ -45,3 +41,32 @@ def test_backoff_strategy():
 
         # the number of retries varies
         assert len(mock.request_history) > 1 
+
+
+def test_backoff_strategy_logs_retries(caplog):
+    caplog.set_level(logging.INFO)
+    filename = "README.md"
+    backoff_strategy = BackoffStrategy(
+        initial_interval=100, max_interval=1000, exponent=1.5, max_elapsed_time=3000
+    )
+    retries = RetryConfig(
+        strategy="backoff", backoff=backoff_strategy, retry_connection_errors=True
+    )
+    
+    with requests_mock.Mocker() as mock:
+        # mock a 500 status code for POST requests to the api 
+        mock.post("https://api.unstructured.io/general/v0/general", status_code=500)
+        session = UnstructuredClient(api_key_auth=FAKE_KEY)
+
+        with open(filename, "rb") as f:
+            files=shared.Files(
+                content=f.read(),
+                file_name=filename,
+            )
+
+        req = shared.PartitionParameters(files=files)
+        with pytest.raises(Exception):
+            session.general.partition(req, retries=retries)    
+
+    
+    assert "seconds before retry" in caplog.text
```

Comment:
This assert looks a little skinny. Can you include more of the exact text that will appear? As a reader, I'd like to have a sense of the user experience when this happens, like a line appears in the log that says "Retrying request ... " or whatever they're going to actually see. Best case would be showing the entire and exact log contents. Does it retry more than once for example? You can make the retry interval very short so it doesn't produce an unnecessarily long test duration.

Reply:
I can make it longer. The log just isn't the same every time because the interval between retries is variable (we randomize it to an extent). 
Yah, it does retry more than once. This test uses a customized backoff strategy so that it retries a few times, but not for longer than 30 sec or something.

Reply:
One thing you might consider is filtering the log if it contains other bits and bobs, like:
```python
lines_I_care_about = [line for line in caplog.text.splitlines() if "identifying text" in line]
```
Then you could assert on the count which assures the number of retries you expect, etc.

Reply:
There's another test that tests retries occurs, so I was thinking of just keeping this one to just checking that logging is happening.